tracing-subscriber/src/fmt/mod.rs

```diff
@@ -448,6 +448,23 @@ where
     }
 }
 
+#[cfg(feature = "json")]
+#[cfg_attr(docsrs, doc(cfg(feature = "json")))]
+impl<T, F, W> SubscriberBuilder<format::JsonFields, format::Format<format::Json, T>, F, W> {
+    /// Sets the json subscriber being built to flatten event metadata.
+    ///
+    /// See [`format::Json`]
```

Comment:
does this link work? i think it needs a reference.

Reply:
no it doesn't :P let me fix that real quick!

Reply:
Ok now it should be ready found another broken link that is now fixed!